build.sbt

```diff
@@ -235,6 +244,17 @@ ThisBuild / publishMavenStyle := true
 ThisBuild / bintrayOrganization := Some("projectglow")
 ThisBuild / bintrayRepository := "glow"
 
+val stableVersion = settingKey[String]("Stable version")
+ThisBuild / stableVersion := IO.read((ThisBuild / baseDirectory).value / "stable-version.txt").trim()
+
+lazy val stagedRelease = (project in file("core/src/test")).settings(
+  commonSettings,
+  resourceDirectory in Test := baseDirectory.value / "resources",
+  scalaSource in Test := baseDirectory.value / "scala",
+  libraryDependencies ++= providedDependencies ++ testDependencies :+ "io.projectglow" %% "glow" % stableVersion.value,
+  resolvers += "bintray-staging" at "https://dl.bintray.com/projectglow/glow"
```

Comment:
What is the priority here? I think we'd want to make sure that the bintray repo has priority over a possible glow artifact that could exist upstream. (Although I guess it is an error anyway to stage an already released version).

Reply:
Good call, I'll just set it to have a single resolver.